common/src/main/java/org/opensearch/ml/common/transport/model/MLModelGetAction.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.model;
+
+import org.opensearch.action.ActionType;
+
+public class MLModelGetAction extends ActionType<MLModelGetResponse> {
+    public static final MLModelGetAction INSTANCE = new MLModelGetAction();
+    public static final String NAME = "cluster:admin/opensearch/ml/models";
```

Comment:
Append "get" to action name? "cluster:admin/opensearch/ml/models/get"
We will have other model actions lile `delete`

Reply:
cool will do.